specification/mediaservices/resource-manager/Microsoft.Media/preview/2019-09-01-preview/MediaGraphs.json

```diff
@@ -0,0 +1,757 @@
+      "post": {
+        "summary": "Start a Media Graph",
+        "description": "Start a Media Graph in the Media Services account",
+        "operationId": "MediaGraphs_Start",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Start Media Graph": {
+            "$ref": "examples/media-graph-start.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/stop": {
+      "post": {
+        "summary": "Stop a Media Graph",
+        "description": "Stop a Media Graph in the Media Services account",
+        "operationId": "MediaGraphs_Stop",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Stop Media Graph": {
+            "$ref": "examples/media-graph-stop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/operationsStatus/{operationId}": {
```

Comment:
Looks like this is to retrieve result of long running operation, if so usually we don't include this operation instead annotate actual LRO operation with `x-ms-long-running-operation`.

Reply:
Both long running operations use "x-ms-long-running-operation": true.  See ling 560 and 608. 
As part of long running operation, we return operationId in response header. User can use this API to get operation status.